.github/workflows/rust-sdk-release.yml

```diff
@@ -0,0 +1,39 @@
+name: Bump Spin Rust Templates SDK Dependency
+
+on:
+  repository_dispatch:
+    types:
+      - rust-sdk-release
+
+jobs:
+  create-pr:
+    name: Create PR with Spin Rust Templates SDK Dependency Bump
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Bump Rust Templates SDK Dependency
+        shell: bash
+        run: ./scripts/bump-rust-template-sdk.sh ${{ github.event.client_payload.version }}
+
+      - name: Import GPG key
+        uses: crazy-max/ghaction-import-gpg@v6
+        with:
+          gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
```

Comment:
Smol nit on the secret keys in this workflow.  I know we('ve) use(d) these same names before, but wondering if we should take the chance to rename to be more helpful in determining their use at a glance?  What do you think about:

`SPINFRAMEWORKBOT_GPG_PRIVATE_KEY`
`SPINFRAMEWORKBOT_GPG_PRIVATE_KEY_PASSPHRASE`
`SPINFRAMEWORKBOT_PAT` <-- not sure if we should make this even more specific for this workflow if we might need different PATs w/ different perms in the future, though this is probably over-thinking; we can always revisit as needed.

Reply:
Or `PULL_REQUESTER` (or similar) if `SPINFRAMEWORKBOT` is too specific (eg maybe for testing on forks 😉)?

Reply:
I agree but would vote for doing this as a follow up; would you mind filing an issue?